examples/ordering/run.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+cargo build --release
+
+clear
+
+for i in {0..4}; do
```

Comment:
Maybe we could make this script also dependent on a number of participants?

Reply:
I'd rather leave it this way - the example requires that we specify the ports manually, and I prefer the slightly weaker assumption "ports 43000-43004 are free" rather than "ports 43000-(43000+n-1) are free"...

Reply:
on the other hand, fixed... (also, see changes in README)